ci/scripts/check-bazel-tags.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+untagged=$(./bazelisk.sh query "rdeps(//..., //hw:verilator) except attr(tags, '[\\[ ](verilator|manual)[,\\]]', //...)")
+if [[ $untagged ]]; then # Check that all targets that depend on verilator are tagged
```

Comment:
nit: just for consistency (also applies to the `if` statements below)

```suggestion
if [[ ${untagged} ]]; then # Check that all targets that depend on verilator are tagged
```